simulator/map_to_schema.py

```diff
@@ -0,0 +1,102 @@
+def MapVMSampleToSchema(VMSample):
+
+    if VMSample:
```

Comment:
What are you trying to detect in this if condition? Make it explicit.

Reply:
checking if vm_sample is None, have made it explicit now.